src/ja/common/proxy/command_handler.py

```diff
@@ -71,6 +74,7 @@ def main_loop(self) -> None:
         while self._running:
             connection, client_address = named_socket.accept()
             try:
+                logger.info("handling command")
```

Comment:
I think it would make sense to print the command that we receive, not just "handling command". So you could print the type of the received command as info and the command itself as debug.